config.toml

```diff
@@ -0,0 +1,3 @@
+database_root="C:/Users/blom_lk/dev/Workspace_FloodAdapt/Database"
```

Comment:
Try to keep personal paths out of the repo. If you want to give an example of what it could/should look like, put that in the readme